### PR TITLE
[MSYS-620]Aws vpc peering connection

### PR DESCRIPTION
**Pull request by harikesh-kolekar**

### CODE REVIEW

File: spec/integration/aws_dhcp_options_spec.rb [hunk 1]
```diff
@@ -13,13 +13,13 @@
...
           end
-        }.to create_an_aws_dhcp_options('test-dhcp-options', configuration: {
-          domain_name: 'example.com',
-          domain_name_servers: %w(8.8.8.8 8.8.4.4),
-          ntp_servers: %w(8.8.8.8 8.8.4.4),
-          netbios_name_servers: %w(8.8.8.8 8.8.4.4),
-          netbios_node_type: 2
-        }).and be_idempotent
+        }.to create_an_aws_dhcp_options('test-dhcp-options', dhcp_configurations: [
+          {:key=>"domain-name", :values=>[{:value=>"example.com"}]},
```
Comment (NAshwini) [inline spec/integration/aws_dhcp_options_spec.rb]: Please use appropriate hash syntax while passing values to dhcp_configurations
eg: use {key: "domain-name", values: [{value: "example.com"}]},

Comment (NAshwini) [inline spec/integration/aws_dhcp_options_spec.rb]: As per our discussion can we please verify the working of `vpc_peering_connection_id` attribute as it was not covered in specs..

Comment (stevendanna) [inline spec/integration/aws_dhcp_options_spec.rb]: [minor] Can we use the more modern hash style here: 

```
{ key: "domain-name",  values: [{value: "example.com"}]}
```

**Review by stevendanna — COMMENTED** [review]
👍 This looks reasonable.  I haven't tested it with v2 myself so I just reviewed for general ruby style.

**Pull request merged**